src/main/java/org/hibernate/models/internal/AnnotationTargetSupport.java

```diff
@@ -118,7 +118,7 @@ default <X extends Annotation> void forEachAnnotationUsage(Class<X> type, Consum
 	default <A extends Annotation> List<AnnotationUsage<? extends Annotation>> getMetaAnnotated(Class<A> metaAnnotationType) {
 		final List<AnnotationUsage<?>> usages = new ArrayList<>();
 		forAllAnnotationUsages( (usage) -> {
-			final AnnotationUsage<? extends Annotation> metaUsage = usage.getAnnotationDescriptor().getAnnotationUsage( metaAnnotationType );
+			final Annotation metaUsage = usage.getAnnotationType().getAnnotation( metaAnnotationType );
```

Comment:
Ahhh.  I initially had all Hibernate and JPA annotations not collect their annotations.  I had to add special handling for certain Hibernate ones.  I just missed this one.

I think this solution is better than my "special handling".